docker/compute_worker/compute_worker.py

```diff
@@ -296,27 +310,41 @@ def _get_bundle(self, url, destination):
                             value["continue"] = False
                     except asyncio.TimeoutError:
                         continue
-
-            end = time.time()
-
-            if kind == 'program':
-                self.program_exit_code = proc.returncode
-                self.program_elapsed_time = end - start
-            elif kind == 'ingestion':
-                self.ingestion_program_exit_code = proc.returncode
-                self.ingestion_elapsed_time = end - start
-
-            logger.info(f'[exited with {proc.returncode}]')
-            for key, value in logs.items():
-                if value["data"]:
-                    logger.info(f'[{key}]\n{value["data"]}')
-                    self._put_file(value["location"], raw_data=value["data"])
-
-            logger.info("Program finished")
-            if self.detailed_results_url and kind == 'program':
-                # Only the scoring program should be able to tell the watcher we are done.
-                self.watch = False
-            return proc
+            except (socket.gaierror, websockets.WebSocketException, ConnectionRefusedError):
```

Comment:
still received this error when restarting django
 `websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason`

Reply:
should be fixed now